views/instruction_view.py

```diff
@@ -0,0 +1,25 @@
+import tkinter as tk
+import tkinterweb
+
+from controls.page_button import PageButton
+from views.base_view import BaseView
+
+
+class InstructionView(BaseView):
+    def __init__(self, controller):
+        super().__init__(controller)
+
+    def __add_widgets(self):
+        back_button = PageButton(
+            self, text="Home", command=self.controller.back_to_main)
+        frame = tkinterweb.HtmlFrame(self, messages_enabled=False)
+
+        # load a website
+        frame.load_website(
+            "https://github.com/PZ-Tetris/tetris/blob/master/README.md")
```

Comment:
it's looking bad

Reply:
I know that, when we will have final docs with instruction I will host static page with limited styles so that it will work well then :)